loadtest/results/frame-path-overhead-2026-08-23/run_arms.py

```
@@ -509,8 +665,27 @@ def main():
+        REMOTE_SSH_ARGV = shlex.split(a.remote_ssh)
+        AI = f"http://{REMOTE_TARGET}:{HTTP_PORT}"
+        PY = a.python_bin or "(로더 박스 인터프리터 — 부하기 실행용, 서버는 --remote-python)"
+        if not a.python_bin:
+            # 로더 쪽도 overhead_rig.py 를 돌릴 인터프리터가 필요하다 — venv 탐색은 그대로 쓴다.
+            PY = resolve_python(a.python_bin)
+        # 🔴 (2026-09-02, R10-b 리허설에서 실측) `--bind` 기본값(127.0.0.1)은 R10-a(동거)
+        #    조건이다 — 원격 모드에서 그대로 두면 대상이 자기 자신에게만 응답해서 로더의
+        #    `wait_health()` 가 전부 실패한다("기동 실패"로만 보여 원인이 안 드러난다).
+        #    사용자가 --bind 를 직접 안 줬을 때만 0.0.0.0 으로 올려 잡는다.
+        if a.bind == "127.0.0.1":
+            a.bind = "0.0.0.0"
```

Ruff (0.16.3)

[error] Possible binding to all interfaces

(S104)